app/controllers/api/account.php

```diff
@@ -1072,39 +1097,40 @@
 
         $queueForEvents->setPayload(
             $response->output(
-                $token->setAttribute('secret', $loginSecret),
+                $token->setAttribute('secret', $tokenSecret),
                 Response::MODEL_TOKEN
             )
         );
 
         // Hide secret for clients
-        $token->setAttribute('secret', ($isPrivilegedUser || $isAppUser) ? $loginSecret : '');
+        $token->setAttribute('secret', ($isPrivilegedUser || $isAppUser) ? $tokenSecret : '');
 
         $response
             ->setStatusCode(Response::STATUS_CODE_CREATED)
             ->dynamic($token, Response::MODEL_TOKEN)
         ;
     });
 
-App::put('/v1/account/sessions/magic-url')
-    ->desc('Create magic URL session (confirmation)')
-    ->groups(['api', 'account', 'session'])
-    ->label('scope', 'public')
-    ->label('event', 'users.[userId].sessions.[sessionId].create')
-    ->label('audits.event', 'session.update')
+App::put('/v1/account/sessions/token')
+    ->alias('/v1/account/sessions/magic-url')
+    ->alias('/v1/account/sessions/phone')
```

Comment:
we might have gotten rid of aliases 🧐

Reply:
The tests are working for me right now?

Reply:
@TorstenDittmann, did we get rid of aliases?